app/Http/routes.php

```diff
@@ -34,5 +34,5 @@
     Route::put('update-student-profile/{remember_token}/{id}','UsersController@adminUpdateStudentProfile');// Admin can updateStudent profile
     Route::put('update-teacher-profile/{remember_token}/{id}','UsersController@adminUpdateTeacherProfile');//Admin can update Student profile
     Route::post('applyforleave/{remember_token}','UsersController@studentApplyForLeave');//Student can apply for date
-
+    Route::put('applyforleave/{id}','UsersController@leaveApproveal');
```

Comment:
try to add user friendly routes not camel case
 Route::put('update-teacher) like this